File: code_blocks/integrations/third-party-integrations/airship_5.dart
import 'package:purchases_flutter/purchases_flutter.dart';
import 'package:airship_flutter/airship_flutter.dart';

Future<void> identifyAirshipUser() async {
  await Purchases.configure(
    PurchasesConfiguration('public_sdk_key')..appUserId = 'my_app_user_id',
  );
  await Airship.identify('my_app_user_id');
}

File: code_blocks/integrations/third-party-integrations/airship_5.ts
import Purchases from '@revenuecat/purchases-react-native';
import Airship from 'urbanairship-react-native';

async function identifyAirshipUser() {
  await Purchases.configure({ apiKey: 'public_sdk_key', appUserID: 'my_app_user_id' });
  await Airship.identify('my_app_user_id');
}
